sections/footer.liquid

```diff
@@ -90,6 +90,12 @@
       "type": "checkbox",
       "id": "show_copyright",
       "label": "Show copyright"
+    },
+    {
+      "type": "text",
+      "id": "privacy_policy_link_text",
+      "label": "Privacy policy link text",
+      "default": "Privacy policy"
```

Comment:
Moved the privacy policy link text to the footer section because it's more appropriate.